module_utils/helpers.py

```diff
@@ -661,6 +661,22 @@ def get_topology_checksum(console_sock, stateboard):
     ''')
 
 
+def enrich_replicasets_with_leaders(control_console, replicasets):
+    leaders, err = control_console.eval_res_err('''
+        return require('cartridge.failover').get_active_leaders()
```

Comment:
Does it work for any failover mode? (Just a question, I'm not sure)

Reply:
Yes